playground/nextjs-app-router/onchainkit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coinbase/onchainkit",
-  "version": "0.34.1",
+  "version": "0.35.0",
```

Comment:
Need to update the `bun.lockb` file - we can just merge this pr though which has the version bump https://github.com/coinbase/onchainkit/pull/1493/files